src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
-body {
+$days:
+  "mon" 0,
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
+$month-length: 28, 29, 30, 31;
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  padding: 10px;
+  margin: 500px auto;
+  max-width: 800px;
```

Comment:
This value should be variable.

Reply:
In the readme, I didn't see a requirement that max-width should be variable. Can you show me, please?